WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostSettingsFragment.java

```diff
@@ -219,6 +222,7 @@ public void onFetchError(Exception error) {
                     public void onSettingsUpdated() {
                         // mEditPostActivityHook will be null if the fragment is detached
                         if (getEditPostActivityHook() != null) {
+                            mAnalyticsTrackerWrapper.track(Stat.EDITOR_POST_FORMAT_CHANGED);
```

Comment:
Do we track events that aren't from the user? I am now reconsidering this placement due to the fact that the default post format is being set here. Let me know.

Reply:
Ideally we should track events due to user action. Let's try to think of a better placement.

Reply:
There are certain special events which are not related to user actions. However, `EDITOR_POST_FORMAT_CHANGED` feels like a user action, so I agree with Ashita's comment. 
One example of an event which isn't directly tied to user action would be for example `ENHANCED_SITE_CREATION_BACKGROUND_SERVICE_UPDATED`.

Reply:
Thank you both for clearing this up :) I will make the necessary changes.

Reply:
Removed 9d41fee